tensorflow/core/kernels/mkl/mkl_batch_matmul_helper.h

```diff
@@ -0,0 +1,93 @@
+#if defined(INTEL_MKL)
```

Comment:
nit: copyright and header `#ifndef` guard

Reply:
done